src/reactapp/src/index.jsx

```diff
@@ -1,6 +1,4 @@
-import React from 'react';
-import 'react-app-polyfill/ie11';
-import ReactDOM from 'react-dom';
+import { render } from "preact";
```

Comment:
since we are using an alias in the vite.config.js we should import render from `'react'` right? Also update to single quotes is eslint/prettier standard I think